packages/wallet-connectors/src/WalletConnect.ts

```diff
@@ -38,30 +40,172 @@ import { UnreachableCaseError } from './error';
+} satisfies WalletConnectModalMobileWallet;
+
+const DEFAULT_MOBILE_WALLETS = {
+    [TESTNET.name]: [CONCORDIUM_WALLET_TESTNET, CRYPTO_X_WALLET_TESTNET],
+    [MAINNET.name]: [CONCORDIUM_WALLET_MAINNET, CRYPTO_X_WALLET_MAINNET],
+};
```

Comment:
These constants should go into another file.

Reply:
I don't know if they "should" as this is a subjective matter? 😄 Is there a good argument for one over the other?

I think their current location is pretty great as these are used specifically for walletconnect, which is clear from their location.